pkg/azure/client/mock/gomock_reflect_1503253645/prog.go

```diff
@@ -0,0 +1,72 @@
+package main
```

Comment:
I am pretty sure that this is a left-over of the mock generation process and should not exist/be commited. You can safely delete it.

Reply:
okay